docs/installation/docker.md

```diff
@@ -0,0 +1,109 @@
+# Docker Compose (ISLE)
+
+The ISLE Islandora Enterprise 8 Prototype ([Islandora-Devops/isle-dc](https://github.com/Islandora-Devops/isle-dc)) is a Docker project to manage Islandora 8 infrastructure under Docker using Docker Compose. These instructions cover the basics of using Docker locally to create a development Islandora sandbox. More detailed configuration options are described in the project's [README](https://github.com/Islandora-Devops/isle-dc).
+
+## Why use Docker?
+
+Docker is a way to separate out your site (i.e. all the files and configurations and data that you entered) from the underlying software that runs it (e.g. webserver, database engine, etc). This allows for easier upgrades, faster development, and more flexible deployment. The ISLE project, run by the Islandora Collaboration Group (ICG) was the driving force behing dockerizing Islandora. 
+
+## Requirements
+
+* Docker (version 19.x+)
+
+To see if you have Docker installed, type `docker --version` in a shell. 
+
+## Installing Docker
+
+If you are installing Docker, we recommend using the application [Docker Desktop](https://www.docker.com/products/docker-desktop). It provides a GUI for managing Docker container in Windows and MacOS, along with the Docker engine and suite of command-line tools. Linux users can download the Engine and command-line tools from that same link. 
+
+There is also a legacy project called [Docker Toolbox](https://docs.docker.com/toolbox/overview/) which may be of interest if your machine cannot run Docker Desktop, or if you already have it installed. 
+
+
+[Download Docker](https://www.docker.com/products/docker-desktop) 
+
+
+## Launching Islandora with Docker
+
+In a shell, clone the isle-dc project. In that directory, enter the command `make` (or `make dev` - see note) to build the docker infrastructure. Then use the command `docker-compose up -d` to start the containers.
```

Comment:
Wasn't it `make demo` for the sandbox version. Or am I misremembering? @dannylamb

Reply:
IIRC from Danny there was `make`, `make dev`, and `make prod`. I din't mention prod because i don't know how to document it. However i don't know how these three make options relate to the part of the ISLE-DC readme, which talks about: 

>  setting ENVIRONMENT in the .env file to either demo, local, or custom. The default is demo.

Reply:
Is there a relationship between 'make'/ 'make dev' and the various environment things described there?

Reply:
not relevant tto this; i'll make a ticket.